thirdparty/github.go

```diff
@@ -1099,34 +1052,43 @@ func GetGithubTokenUser(ctx context.Context, token string, requiredOrg string) (
 	}, isMember, err
 }
 
-// CheckGithubAPILimit queries Github for the number of API requests remaining
-func CheckGithubAPILimit(ctx context.Context) (int64, error) {
```

Comment:
I'm seeing that this was used for CheckGithubAPIResources, which idk that we need that, we can probably remove that throughout

Reply:
CheckGithubAPIResources is only used once but it's to check if we have enough limit for repotracker 

I think that could be valuable since it would save us from doing a bunch of work just to run into it later (also its free to check!)